trySwift/MoreTableViewController.swift

```diff
@@ -161,35 +182,10 @@ extension MoreTableViewController {
 
 private extension MoreTableViewController {
     
-    func showAbout() {
-        let aboutViewController = AboutTableViewController()
-        performSegue(withIdentifier: moreDetailSegue, sender: aboutViewController)
-    }
     
-    func showVenues() {
-        let venueController = VenuesViewController()
-        performSegue(withIdentifier: moreDetailSegue, sender: venueController)
-    }
     
-    func showCodeOfConduct() {
-        let webViewController = WebDisplayViewController()
-        webViewController.url = URL(string: conference.codeOfConductURL)!
-        webViewController.displayTitle = "Code of Conduct".localized()
-        performSegue(withIdentifier: moreDetailSegue, sender: webViewController)
-    }
-    
-    func showOrganizers() {
-        let organizerViewController = OrganizersTableViewController()
-        performSegue(withIdentifier: moreDetailSegue, sender: organizerViewController)
-    }
-    
-    func showLibraries() {
-        let path = Bundle.main.path(forResource: "Pods-trySwift-acknowledgements", ofType: "plist")
-        let acknowledgementesViewController = AcknowListViewController(acknowledgementsPlistPath: path)
-        acknowledgementesViewController.edgesForExtendedLayout = []
-        acknowledgementesViewController.headerText = "We ❤️ Open Source Software".localized()
-        
-        performSegue(withIdentifier: moreDetailSegue, sender: acknowledgementesViewController)
+    func navigateTo<T: UIViewController>(id: String = "moreDetailSegue", _ controller: () -> T){
```

Comment:
🤔, I don't really like this signature. Especially when used without spacing for the `{}`'s. What about `func navigate<T: UIViewController>(to controller: () -> T, id: String = "moreDetailSegue")`? Then use it with the parameter:

```swift
navigate(to: { acknowledgement })
```

Thoughts, @NatashaTheRobot?

Reply:
Now that I think of it... is there any reason why this couldn't just accept a `UIViewController` instead of a closure?

Reply:
Looks better indeed, I've replaced it